coverage.py

```diff
@@ -341,11 +341,45 @@ def work(self, identifier):
             return CoverageFailure(self, identifier, e, transient=True)
         return work
 
-    def set_metadata(self, identifier, metadata, 
+
+    def set_meta_circ_data(self, identifier, metadata, circulationdata, 
```

Comment:
Sure!  I was trying to curb my long-method-name enthusiasm for the team's sake :).